src/main/java/com/nike/cerberus/service/MetadataService.java

```diff
@@ -151,6 +152,19 @@ public String getSdbId(SDBMetadata sdbMetadata) {
         return id;
     }
 
+    /**
+     * Overrides the owner of the SDB. This is an admin function.
+     * @param name the name of the SDB
+     */
+    public void overrideOwner(String name, String newOwner, String adminUser) {
```

Comment:
name, seems like a bad variable name in this instance, when it is really the sdbName. We have multiple things in this code base that have name. immediately looking at the method name I thought maybe it was the owner name. Looking at the java doc, I can finally see what it is for.

Also the Javadoc is missing params.

Reply:
- [ ] self describing variable name
- [ ] fix java do

Reply:
updated